build.gradle

```diff
@@ -13,6 +13,9 @@ subprojects {
         maven { url = 'https://repo.codemc.org/repository/maven-public/' }
     }
 
+    defaultTasks 'build', 'shadowJar'
```

Comment:
Is shadowJar needed here given the modification https://github.com/knightzmc/PrivateMines/pull/33/files#diff-520f178549fabd7e74f5b1fedb7275a1434a2ac153fc49e0212455ceeef38e37R19 ?

Reply:
Without it I wasn't getting the jar using the command gradlew build as suggested on instructions to build